infra/dockerfile.base

```diff
@@ -1,13 +1,13 @@
-FROM python:3.7.3-slim
+FROM python:3.7.3-alpine
 
 # Setup dependencies in a cacheable step
 COPY requirements.txt /requirements.txt
-RUN apt-get update && \
-		apt-get install -y --no-install-recommends gcc g++ libgomp1 && \
+
+RUN apk update && apk upgrade && \
+		echo "manylinux1_compatible = True" > /usr/local/lib/python3.7/site-packages/_manylinux.py && \
+		apk add --no-cache gcc g++ libgomp lapack-dev git libexecinfo-dev make libc6-compat && \
```

Comment:
You are not removing these packages anymore. This is going to increase the size of the resulting image.

Reply:
Sorry, dropped out of my mind. Updated PR